README.md

```diff
@@ -165,7 +165,19 @@ We offer the following installation methods:
 > 💡 Recommended if you're using Google Cloud Platform and want to try it on
 > a realistic cluster.
 
-1.  Install tools specified in the previous section (Docker, kubectl, skaffold)
+
+1. Install tools to run a Kubernetes cluster locally:
+
+   - kubectl (can be installed via `gcloud components install kubectl`)
+   - Local Kubernetes cluster deployment tool:
+        - [Minikube (recommended for
+         Linux)](https://kubernetes.io/docs/setup/minikube/)
+        - [Docker for Desktop (recommended for Mac/Windows)](https://www.docker.com/products/docker-desktop)
+          - It provides Kubernetes support as [noted
+     here](https://docs.docker.com/docker-for-mac/kubernetes/)
+        - [Kind](https://github.com/kubernetes-sigs/kind)
+   - [skaffold]( https://skaffold.dev/docs/install/) ([ensure version ≥v1.10](https://github.com/GoogleContainerTools/skaffold/releases))
+
```

Comment:
I'm not sure duplicating this with above is a great idea. Almost certainly the next PR will change only one of the duplicates; not both. :)

Reply:
What about moving the "install tools" section to a prerequisites section at the top of the doc?

Reply:
yeah would work 👍